deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,252 @@
+; Shows a lot of debug information while compiling
+; This can be removed once stable.
+!verbose 4
+;--------------------------------
+
+    !include "MUI2.nsh"
+	
+    Var _JELLYFINVERSION_
+    Var _JELLYFINDATADIR_
+    Var _SERVICEINSTALLED_
+	Var _EXISTINGINSTALLATION_
+	Var _EXISTINGSERVICE_
+
+;--------------------------------
+;General
+
+; Align installer version with jellyfin.dll version
+    !getdllversion "$%InstallLocation%\jellyfin.dll" ver_
+    !echo "jellyfin.dll version is ${ver_1}.${ver_2}.${ver_3}.${ver_4}" ;!echo will print it while building
+
+    Name "Jellyfin ${ver_1}.${ver_2}.${ver_3}.${ver_4}"
```

Comment:
This might want a `Server` suffix, to avoid confusion with potential clients.

Reply:
Agree.